scripts/test/test-utils.sh

```diff
@@ -57,16 +57,16 @@ convert_and_test() {
     ${MENDER_CONVERT_EXTRA_ARGS}
 
   local ret=0
-  run_tests "${device_type}" "${artifact_name}" || ret=$?
+  run_tests "${device_type}" "${image_name%.img}-mender" || ret=$?
```

Comment:
Could you rename this variable to `image_file`? It's quite confusing to read because in the other file, `image_name` is without suffix, here it is with suffix.

The same should probably be done with `image_name_compressed`, which doesn't appear in your patch, but which is related.